### DISCUSSION TITLE
Can we include all colors (like amber, lightBlue) by default in `jit` mode?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#3999] [Ideas]
Can we include all colors (like amber, lightBlue) by default in `jit` mode?

**Comment by simonswiss** [comment]
Having access to all the colors by default would certainly not be a "bloat" issue anymore with JIT, but one could argue that it "damages" the initial _constraint decisions_ when it comes to colors.

Just like adding too many font size options to the `fontSize` scale would lead to less consistency, having too many "close" colors like `emerald` and `green`, `rose` and `pink`, would  lead to potential sloppiness.

If you know what you're doing, by all means, enable all the colors in the config. But in my opinion, making these available _by default_ could have some negative impact on the _constraint based design_ philosophy of Tailwind CSS.
